### ISSUE TITLE
Adaptive Card links should not be disabled even if the card is disabled

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#3618]
## Version
4.11.0

## Describe the bug

Currently, published behavior is that Web Chat's anchors will not work on first click, but will redirect to the url on second click. The original intention was to disable all element clicks, not just the first one. This behavior needs to be fixed.

When Adaptive Cards are disabled after use (when they are no longer the most recent activity), buttons and inputs should be disabled, but anchors should not. 

Right now the AC renderer will disable the following elements: `<a>`, `<button>`, `<input />`, `<select>`, `<textarea>`. All elements except anchors are reasonable to disable, but anchors should still be accessible even if the rest of the card is not. Since all anchors in Adaptive Cards are treated as links, as opposed to buttons, it's superfluous to disable them.

This bug is two-fold:
1.  Disabling the anchor should be removed
1. The event listener undoing the action for other elements should listen on ALL clicks, not just the first one.

## Steps to reproduce
1. Show a card that contains an anchor
1. Send another activity
1. Click on anchor
1. First click does nothing
1. Second click navigates to url

`attachmentMiddleware`:
```
      var attachmentMiddleware = function () {
        return function (next) {
          return function (card) {
            var attachment = card.attachment;
 
            switch (attachment.contentType) {
              case 'application/vnd.microsoft.card.adaptive':
                return window.React.createElement(window.WebChat.Components.AdaptiveCardContent, {
                  actionPerformedClassName: 'card__action--performed',
                  content: attachment.content,
                  disabled: true
                  // disabled: !recentBotMessage
                });
 
              default:
                return next(card);
            }
          };
        };
      };
```

## Expected behavior
Anchors should not be disabled, and disabled elements's event listener with undo should happen on every subsequent click, not just on the first click.

1. Remove anchors from disabled elements list
1. Add `addEventListenerWithUndo` that listens to all clicks, not just the first (as opposed to `addEventListenerOnceWithUndo`

### Addtional context

Testing scenarios:
1. With AC disabled, make sure links is still working and opening new window
1. Add a new test
   1. Add a new attachment middleware that disable AC, but not hero card
   1. Put a document.onclick handler
   1. Type "card breakfast"
   1. With AC disabled, click on one of those disabled button
   1. Expect, the document.onclick handler is not fired
   1. Type "herocard qna"
   1. Click on one of the button on hero card
   1. Expect, the document.onclick handler is fired once

[Bug]
